Content.Server/CartridgeLoader/Cartridges/NanoTaskCartridgeComponent.cs

```diff
@@ -0,0 +1,36 @@
+using Content.Shared.CartridgeLoader.Cartridges;
+
+namespace Content.Server.CartridgeLoader.Cartridges;
+
+[RegisterComponent]
+public sealed partial class NanoTaskCartridgeComponent : Component
+{
+    /// <summary>
+    /// The list of tasks
+    /// </summary>
+    [DataField]
+    public List<NanoTaskItemAndId> Tasks = new();
+
+    /// <summary>
+    /// counter for generating task IDs
+    /// </summary>
+    [DataField]
+    public int Counter = 1;
+
+    /// <summary>
+    /// Remaining time of printing animation
+    /// </summary>
+    [DataField]
+    public float PrintDelayRemaining = float.NaN;
+
```

Comment:
Is this supposed to be a required field?

Reply:
no